kmod/core/Makefile

```diff
@@ -13,11 +14,18 @@ kpatch.ko: core.c
 
 all: kpatch.ko
 
+install:
+	$(INSTALL) -d $(MODULESDIR)/$(shell uname -r)
+	$(INSTALL) -m 644 core/kpatch.ko $(MODULESDIR)/$(shell uname -r)
+	$(INSTALL) -m 644 core/Module.symvers $(MODULESDIR)/$(shell uname -r)
+
```

Comment:
No need to specify `core/` in the path, since we would already be in the `core/` directory when this target is called. So it'd just  be `$(INSTALL) -m 644 kpatch.ko`